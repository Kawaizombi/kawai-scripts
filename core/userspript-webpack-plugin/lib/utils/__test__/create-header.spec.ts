import createHeader from '../create-header';

const config = {
  name: 'User script',
  noframes: true,
  grant: [
    'GM_getValue',
    'GM_setValue',
  ],
};

describe('createHeader', () => {
  it('should return header', () => {
    expect(createHeader(config)).toMatchSnapshot();
  });
});
