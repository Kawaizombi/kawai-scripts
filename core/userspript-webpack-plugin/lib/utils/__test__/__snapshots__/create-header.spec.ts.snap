// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createHeader should return header 1`] = `
"// ==UserScript==
// @name User script
// @noframes
// @grant GM_getValue
// @grant GM_setValue
// ==/UserScript=="
`;
